src/components/character/Character.js

```diff
@@ -7,13 +7,25 @@ import css from './Character.module.css';
 
 import characters from './characters.json';
 
-const Character = ({ num, className }) => {
+const Character = ({ className }) => {
+  const newVersion = () => Math.floor(Math.random() * 3);
+  const [version, setVersion] = useState(newVersion);
```

Comment:
This will cause a mismatch between the server-rendered markup and the hydrated front-end version, since the random value will be different. I've been working on a `useRandom` hook for Superbright that uses `seedrandom` to ensure random values between server and client, but for now we can just start with the same version number and use random in `useEffect` since that only runs on the client.

Reply:
fixed